### DISCUSSION TITLE
How to configure Dify to disable authentication

### DISCUSSION THREAD

**Discussion by lennonn** [langgenius/dify#5444] [Help]
Other systems can call Dify's API and get results without authentication.

**Comment by perzeuss** [comment]
That is not possible and probably will never be, because the API Key is the app identifier. If you do not submit the API Key, Dify would not know which app you want to communicate with.

**Reply by lennonn** [reply]
if i want to  call this api   https://localhost/console/api/apps?page=1&limit=30&name=&mode=workflow 。Where can I get the api-key？

**Reply by perzeuss** [reply]
That's not a public API, you cannot use it with an API Key. It's the internal API and works with a user auth token.
